pkg/node/node.go

```diff
@@ -138,6 +146,53 @@ func NewBee(addr string, swarmAddress swarm.Address, keystore keystore.Service,
 	b.stateStoreCloser = stateStore
 	addressbook := addressbook.New(stateStore)
 
+	if o.SwapEnable {
+		swapBackend, err := ethclient.Dial(o.SwapEndpoint)
+		if err != nil {
+			return nil, err
+		}
+		transactionService, err := chequebook.NewTransactionService(logger, swapBackend, signer)
+		if err != nil {
+			return nil, err
+		}
+		overlayEthAddress, err := signer.EthereumAddress()
+		if err != nil {
+			return nil, err
+		}
+
+		// print ethereum address so users know which address we need to fund
+		logger.Infof("using ethereum address %x", overlayEthAddress)
+
+		// TODO: factory address discovery for well-known networks (goerli for beta)
```

Comment:
This is important

Reply:
yes, will do this in a separate PR when we deployed the real token and factory.